components/server/src/auth/bearer-authenticator.ts

```diff
@@ -39,9 +37,12 @@ function createBearerAuthError(message: string): BearerAuthError {
     return Object.assign(new Error(message), { code: bearerAuthCode } as { code: typeof bearerAuthCode });
 }
 
+const patPrefix = "gitpod_pat_";
```

Comment:
Some context where this magic comes from would be nice. :sparkle:

Reply:
done.